### PR TITLE
Add settings.yml

### LINKED ISSUE #5414: Use settings.yml
See https://github.com/apps/settings

### PR DESCRIPTION
**Pull request by Pandapip1**
Blockers: #5418, #5400, an admin of @ethereum installing https://github.com/apps/settings for this repository

Closes #5414

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Require a review from these people in order to merge potentially dangerous changes
+.github     @MicahZoltu
```
Comment (Pandapip1) [inline .github/CODEOWNERS:2]:
```suggestion
.github     @SamWilsn
```

File: .github/settings.yml [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    description: This PR has received feedback from an EIP Editor/Author and is awaiting the PR Author to update.
+  - name: waiting: author to review
+    color: CC0000
+    description: This Pull Request changese an EIP and requires review by an author of that EIP.
+  - name: waiting: editor to review
+    color: CC0000
+    description: This Pull Request requires review by an EIP Editor.
+
+collaborators:
+  - username: MicahZoltu
```
Comment (Pandapip1) [inline .github/settings.yml:82]:
```suggestion
  - username: SamWilsn
```

**Comment by lightclient** [comment]
I prefer to not use `settings.yml`. We have had some problems with branch protection in the past and I am not comfortable with someone potentially getting a malicious commit into it.

**Comment by Pandapip1** [comment]
I'll close this for now, until everything has been settled.

**Pull request closed without merging**